src/utils/global/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -7,6 +9,8 @@ import { classes } from '../../classes'
 import styles from './ambassadorCard.module.scss'
 import moderatorBadge from '../../../assets/mod.png'
 
+import Tooltip from '../tooltip/Tooltip'
```

Comment:
Nitpick, this should be sorted above the styles/assets imports, with the chatCommand import etc.